recipes/igblast/build.sh

```diff
@@ -1,36 +1,34 @@
 #!/bin/bash
 
 IGBLAST_ADDRESS=ftp://ftp.ncbi.nih.gov/blast/executables/igblast/release
-INCLUDE_DIR=$PREFIX/include/igblast
+SHARE_DIR=$PREFIX/share/igblast
 
 wget $IGBLAST_ADDRESS/edit_imgt_file.pl
-sed -i '1 s/^.*$/#!\/usr\/bin\/env perl/g' edit_imgt_file.pl # replace the first line with /usr/bin/env perl instead of the hard-coded /opt
+# replace the first line with /usr/bin/env perl instead of the hard-coded /opt
+# does this require an explicit perl dependency? 
+sed -i '1 s/^.*$/#!\/usr\/bin\/env perl/g' edit_imgt_file.pl 
```

Comment:
I always thought that conda is able to fix hardcoded paths in shebang files by itself. Doesn't this work here?
Apart from that, yes, the package should have a perl dependency then. But that is no problem, we already have perl in bioconda.